src/client/main.tsx

```diff
@@ -21,10 +22,12 @@ export class Main extends React.Component<{}, ClientState> {
   socket: SocketIOClient.Socket;
   gameToJoin: string;
   activePlayer: string;
+  isWinner: boolean;
```

Comment:
This might not even be necesssary, can just use `const isWinner = Game.getWinner(this.gameState) === this.activePlayer` in the render function.